neurots/generate/tree.py

```diff
@@ -140,6 +140,7 @@ def _initialize_algorithm(self):
             start_point=self.point,
             context=self.context,
             random_generator=self._rng,
+            pia_direction=self.params.get("pia_direction"),
```

Comment:
I see this is exposed at the tree level only. This should be at the neuron level if the whole neuron is rotated?

Reply:
it is also use to place trunks, but from previous PRs